ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_D.yml

```diff
@@ -58,7 +64,6 @@ Tenant_D:
           name: Tenant_D_v6_WAN_Zone_1
           tags: ['v6wan']
           enabled: True
-          ipv6_address: 2001:db8:350::1/64
```

Comment:
Instead of modifying existing cases, which could lead to loss of coverage, please add new SVIs with clear descriptions testing each case.

Reply:
well now we actually support this, since we just look at the svi